packages/techdocs-cli/src/e2e.test.ts

```diff
@@ -1,99 +1,104 @@
-import { spawn } from 'child_process';
+import { spawn } from "child_process";
 
-describe('end-to-end', () => {
-
-  it('shows help text', async () => {
-    const proc = await executeTechDocsCliCommand(['--help']);
+describe("end-to-end", () => {
+  it("shows help text", async () => {
+    const proc = await executeTechDocsCliCommand(["--help"]);
 
     expect(proc.exit).toEqual(0);
-    expect(proc.combinedStdOutErr).toContain('Usage: techdocs-cli [options]');
+    expect(proc.combinedStdOutErr).toContain("Usage: techdocs-cli [options]");
   });
 
-  it('can serve', async () => {
-    jest.setTimeout(10000);
-    const proc = await executeTechDocsCliCommand(['serve', '--no-docker'], {
-      cwd: '../../',
-      killAfter: 8000,
+  it("can serve", async () => {
+    jest.setTimeout(12000);
+    const proc = await executeTechDocsCliCommand(["serve", "--no-docker"], {
+      cwd: "../../",
+      killAfter: 10000
     });
 
-    expect(proc.combinedStdOutErr).toContain('Serving docs in Backstage at');
+    expect(proc.combinedStdOutErr).toContain("Serving docs in Backstage at");
   });
 
-  it('can serve:mkdocs', async () => {
-    jest.setTimeout(10000);
-    const proc = await executeTechDocsCliCommand(['serve:mkdocs', '--no-docker'], {
-      cwd: '../../',
-      killAfter: 8000,
-    });
-
-    expect(proc.combinedStdOutErr).toContain('Starting mkdocs server on');
+  it("can serve:mkdocs", async () => {
+    jest.setTimeout(12000);
+    const proc = await executeTechDocsCliCommand(
+      ["serve:mkdocs", "--no-docker"],
+      {
+        cwd: "../../",
+        killAfter: 10000
+      }
+    );
+
+    expect(proc.combinedStdOutErr).toContain("Starting mkdocs server on");
```

Comment:
With a stdout/stderr containing `Non zero exit code from child process`, it would be interesting to debug the build and see what the exit code is (e.g. just a `console.log(proc.exit);`).  Maybe also try with `-v` passed as a flag to the command too.